doc/user_guide/Running_in_Webassembly.md

```diff
@@ -403,3 +404,28 @@ HTML('<b>HTML!</b>')
 ### Usage
 
 The code cell will display a button to execute the cell, which will warn about downloading the Python runtime on first-click and ask you to confirm whether you want to proceed. It will then download Pyodide, all required packages and finally display the output.
+
+## Setting up JupyterLite
+
+[JupyterLite](https://jupyterlite.readthedocs.io/en/latest/) is a JupyterLab distribution that runs entirely in the browser built from the ground-up using JupyterLab components and extensions. In order to use Panel in JupyterLite you will have to build your own distribution. As a starting point we recommend following [this guide](https://jupyterlite.readthedocs.io/en/latest/howto/configure/simple_extensions.html) in the JupyterLite documentation which will tell you how to set up an environment to begin building JupyterLite.
```

Comment:
I don't understand "a JupyterLab distribution that runs entirely in the browser built from the ground-up using JupyterLab components and extensions". Aren't all JLab distributions built from the ground up using JLab components and extensions?

Reply:
I guess so but take it up with Jupyterlite folks, this is literally just copying their own description.

Reply:
Well, having copied it, you're responsible for it now! So it would be good to restate it using words that we all understand. E.g. my guess would be something like "JupyterLite is a JupyterLab distribution built from all the usual components and extensions that come with JupyterLab, but now running entirely in the browser with no external server needed".